webapp/src/pages/offer/components/OfferComponent.js

```diff
@@ -31,30 +30,43 @@ class Offer extends Component {
     };
   };
 
-  buttonSubmit() {
-    const { offer, accepted, rejected, rejected_reason } = this.state;
+  buttonSubmit(candidate_response) {
+    const { offer, rejected_reason } = this.state;
 
     offerServices
       .updateOffer(
         offer.id,
         DEFAULT_EVENT_ID,
-        accepted,
-        rejected,
-        rejected_reason
+        candidate_response,
+        candidate_response? "" : rejected_reason
       )
       .then(response => {
         if (response.msg === "succeeded") {
           this.setState({
             offer: response.data
           });
+          this.displayOfferResponse();
         } else if (response.error) {
           this.setState({
             error: response.error
           });
         }
       });
   }
-
+  
+  displayOfferResponse = e =>{
+       const { offer } = this.state;
+       return (<div className="container">
+           <div className="white-background card form">
+              {offer.candidate_response?
+              "Accepted an offer"
```

Comment:
Can we update this to be nice?

"You have accepted the following offer:
insert details here

"You have rejected the following offer:
insert offer details here
reason